docs/introduction/computational_resources_overview.md

```diff
@@ -58,13 +45,11 @@ that exceed the capabilities of a single machine.
 
 ## Comparison Example: SWASH Simulation
 
-In this comparison, we aim to illustrate the performance differences between 
-running simulations on a shared pool of resources and on dedicated resources using the 
-Inductiva API. 
-
-We will use the [SWASH simulation]() as our example to demonstrate these differences.
+To illustrate the performance differences between running simulations on a shared 
+pool of resources and on dedicated resources using the Inductiva API, we will run 
+the [SWASH simulation]() as an example.
```

Comment:
where should this link to?

Reply:
either our own link to the swash in the simulator section, or the repo of the peeps behind swash? https://gitlab.tudelft.nl/citg/wavemodels/swash

Reply:
I think it should be an internal link to our SWASH page, but it would be fine to add the link of the developers from our internal swash page.

Reply:
I think it should point to the SWASH simulator, but just the SWASH term `[SWASH](../simulators/SWASH)`